pagopa/io-app
# fix: [IOPID-2389] - Fix login CieID

## Short description
This PR resolves the issue related to partial login with CieID, where the session was lost once the app was closed.

## List of changes proposed in this pull request
- In `ts/hooks/useNavigateToLoginMethod.tsx`:
    - Removed memoization in the `isCieSupported` flag
    - Added `dispatch(idpSelected(IdpCIE));` before navigating in the `CieIdLoginScreen` to properly complete the login flow
- Removed unnecessary comment from `ts/screens/profile/PrivacyMainScreen.tsx`

## Demo
|Before    |After       |
|---------|---------|
|<video src="https://github.com/user-attachments/assets/94799f02-90ad-4990-9d6e-66e4d50eb6f8" />|<video src="https://github.com/user-attachments/assets/10e5da5a-c066-4530-ab7b-b2be8efdb76c" />|

## How to test
Log in using `CieID`, once you have entered the app, close and reopen it; you should still be authenticated.
